src/locales/locale-en.json

```diff
@@ -364,6 +364,15 @@
     "EDIT_ACTIVITY_TYPES": "Activity types",
     "EDIT_DETAILS": "Group details",
     "EMAIL_NOTIFICATIONS": "Email notifications",
+    "EXTRA_INFO": {
+      "LOCATION": "If you want your group's location to be made visible on a public map, enter an adress or pick a location on the map below.",
+      "NEW_MEMBERS": "New members will have to apply and get accepted to your group. You can always change the application and welcome message later in the group settings."
```

Comment:
We have a loose convention to put these hints next to the text they are a hint about with `_HINT` added to the key.

So in this case they would be:
- `GROUP.HEADINGS.LOCATION_HINT`
- `GROUP.HEADINGS.NEW_MEMBERS_SIGNUP_HINT`

Reply:
Cool, I was actually wondering how I should go about when doing this. Now I know!
Updated.